scripts/commands/speed.lua

```diff
@@ -15,14 +16,20 @@ function error(player, msg)
 end
 
 function onTrigger(player, speed)
+    if not speed then
+        player:PrintToPlayer(string.format("Current Speed: %u", player:speed()))
+        player:PrintToPlayer(string.format("Current tpz.mod.MOVE: %u", player:getMod(tpz.mod.MOVE)))
+        return
+    end
 
-    -- validate speed amount
-    if (speed == nil or speed < 0 or speed > 255) then
+    -- Validate speed amount
+    if speed < 0 or speed > 255 then
         error(player, "Invalid speed amount.")
         return
     end
 
-    -- set speed
-    player:speed( speed )
-
+    -- Inform player and set speed
+    player:PrintToPlayer(string.format("Old Speed: %u", player:speed()))
+    player:PrintToPlayer(string.format("New Speed: %u", speed))
+    player:speed(speed)
```

Comment:
I... can't believe this is the original binding... would you mind swooping in and changing this to `setSpeed()` or something better?

If it's in 10 million places don't worry about it, it isn't _that_ awful 🙏

Reply:
There were actually a lot of the old bindings that "report current" if no input was given, with no get/set. Pretty much anytime it was a member value instead of a function call..

Do we want 2 bindings? They would both be acting on `m_baseentuty->speed`
https://github.com/project-topaz/topaz/blob/release/src/map/lua/lua_baseentity.cpp#L5255

A lot about speed need reworked and I thought I'd leave this alone until we do that. This is the tip of the iceberg and it is all ugly imo.

Reply:
Yeah, just leave it be for now then 👍